### PR TITLE
 Replace $verbose with VERBOSITY_VERBOSE

### PR DESCRIPTION
**Pull request by kesselb**
`--verbose` and `--quiet` works out of the box for commands. When you are fine with the default behaviour there is no need to redefine these options within an command.

https://github.com/nextcloud/3rdparty/blob/f48c7e29b8001dff95ac00c16338becef1f175dd/symfony/console/Application.php#L865-L868

This pr replace `if ($verbose)` with `OutputInterface::VERBOSITY_VERBOSE`

### CODE REVIEW

**Comment by danielkesselberg** [comment]
I'm a bit unsure about this one

https://github.com/nextcloud/server/blob/0fca815b4dd601381a909656a1e3e7a7e89714d2/apps/files/lib/Command/Scan.php#L206-L209

suppress output  looks wrong to me but it was there before.

**Comment by ChristophWurst** [comment]
Nice! Didn't know about this flag. Will definitely use this in future commands :rocket:

**Comment by ChristophWurst** [comment]
> I'm a bit unsure about this one

Maybe @icewind1991 can give some input on this. Ref https://github.com/nextcloud/server/commit/ce2dba079605204293373652a53627eb50dd39d7.

File: apps/files/lib/Command/Scan.php [hunk 1]
```diff
@@ -120,39 +108,32 @@ public function checkScanWarning($fullPath, OutputInterface $output) {
...
-				$this->abortIfInterrupted();
-			});
-			$scanner->listen('\OC\Files\Utils\Scanner', 'scanFolder', function () use ($output) {
-				$this->foldersCounter += 1;
-				$this->abortIfInterrupted();
-			});
-		}
+
+		$scanner->listen('\OC\Files\Utils\Scanner', 'scanFile', function ($path) use ($output) {
+			$output->writeln("\tFile\t<info>$path</info>", OutputInterface::VERBOSITY_VERBOSE);
```
Comment (danielkesselberg) [inline apps/files/lib/Command/Scan.php]: @ChristophWurst may i revert `\t` like you suggested in https://github.com/nextcloud/server/pull/11968?

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: apps/files/lib/Command/Scan.php (hunk 1 above)
Comment (MorrisJobke) [reply apps/files/lib/Command/Scan.php]: I'm fine with this.

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**